webpack.dev.js

```diff
@@ -6,6 +6,9 @@ const HOST = process.env.HOST || "localhost";
 const PORT = process.env.PORT || "9000";
 
 module.exports = merge(common('development'), {
+  entry: {
+    app: ['react-hot-loader/patch', path.resolve(__dirname, 'src', 'index.tsx')]
+  },
```

Comment:
This has always been buggy at best and can be confusing when you change a component that is in a certain state. Can you just remove it?

Reply:
Removing the HMR entirely? Yep can do, I didn't have much luck with it in my app either

Reply:
Yep! Remove the config and the package please.

Reply:
HMR related packages, usage, and config params removed!